### ISSUE TITLE
Guard let else return as a one liner?

### ISSUE THREAD

**Issue by mokagio** [iflix-letsplay/swift-style-guide#9]
In the codebase we're using two different styles of early returns:

```swift
guard condition else { return }
```

and 

```swift
guard condition else {
  return
}
```

We should pick one and stick to it.

**Comment by mokagio** [comment]
+1 this for the oneliner

**Comment by mokagio** [comment]
+1 this for the... _newliner_ (?)

**Comment by cuteiosdev** [comment]
I think there's a bit of nuance here. When we need to do something as well as return, I think it should be the newline option. When it's just return, it should be the same line as the last guard statement.

**Comment by TimKregerIflix** [comment]
Although if we go with newliner it removes the nuance.

**Comment by adriaan** [comment]
In defence of newliner: it does make it easier to set breakpoints. (though you could always make it a newliner temporarily if you need to set a breakpoint).

**Comment by TimKregerIflix** [comment]
Yeah thats my primary reason @adriaan it just makes it easier when doing complex traces

**Comment by mokagio** [comment]
> I think there's a bit of nuance here. When we need to do something as well as return, I think it should be the newline option. When it's just return, it should be the same line as the last guard statement.

Sure. The code example only has `{ return }` and that was what the question was about. Although I didn't state anywhere 😳 .
